README.md

```diff
@@ -110,8 +110,11 @@ In this case, please install vLLM first. You can do so by running
 container with the following commands:
 
 ```
-mkdir -p /opt/tritonserver/backends/vllm
-wget -P /opt/tritonserver/backends/vllm https://raw.githubusercontent.com/triton-inference-server/vllm_backend/main/src/model.py
+vllm_tmp_dir=/tmp/backends/vllm_backend
+mkdir -p /opt/tritonserver/backends/vllm $vllm_tmp_dir
+git clone https://github.com/triton-inference-server/vllm_backend.git $vllm_tmp_dir
+cp -r $vllm_tmp_dir/src/* /opt/tritonserver/backends/vllm
+rm -rf $vllm_tmp_dir
```

Comment:
```suggestion
TEMP_TRITON_VLLM_DIR=/tmp/vllm_backend
mkdir -p /opt/tritonserver/backends/vllm $vllm_tmp_dir
git clone https://github.com/triton-inference-server/vllm_backend.git $TEMP_TRITON_VLLM_DIR
cp -r $TEMP_TRITON_VLLM_DIR/src/* /opt/tritonserver/backends/vllm
rm -rf $TEMP_TRITON_VLLM_DIR
```

Reply:
in current version you create `backends/vllm_backend` path, but delete only `vllm_backend` , `backends` folder is still left

Reply:
If `/tmp/backends` exists we shouldn't delete files other than vllm_backend in this directory.

Reply:
Added `rm -d /tmp/backends` to prevent unexpected removal of files.

Reply:
On second thought, I think it's fine to clone into `/tmp/vllm_backend`.